### DISCUSSION TITLE
GraphQL support & Future of the project

### DISCUSSION THREAD

**Discussion by kawpii** [daptin/daptin#125] [Q&A] [answered]
First of all cool project! Thanks for great work!

Now to the questions

- Is provided GraphQL support full featured?
- Does Daptin JS SDK support GraphQL? (Can I use it with Gatsbyjs)
- Do you plan to keep maintaining the project for a foreseeable future?

Thanks in advance!

**Comment by artpar** [comment, marked as answer]
Hey @kaushalyap 

1. GraphQL support is actually secondary and not primary. JSON API schema support is the primary target. At present all the set of features exposed on JSON API are available on graphql as well. Let me know if you are looking into something specific.

2. It should work, if you face any issues please create an issue

3. Yes, daptin is used across a wide variety of projects and I will maintain it at least for the foreseeable future. Although I am not planning to add any more big features (caldav/carddav is going to be the last major feature which is being worked on by a separate maintainer).
